### DISCUSSION TITLE
Maybe current Folder & Dashboard selection could be better ?

### DISCUSSION THREAD

**Discussion by Joey777210** [grafana/grafana#68943] [General]
Here is some observe:
1. There is no quick way to select most of the dashboards. If I select a folder, all the dashboards inside it will be selected, and then I want to deselect some of them and do something with the remaining ones, it doesn't seem possible at the moment.Caz the folder will still be selected, and if I deselect the folder, all dashboards will be deselected.
So if I wanna select a lot dashboards, I had to click one by one.
![Kapture 2023-05-24 at 14 55 26](https://github.com/grafana/grafana/assets/53630996/f121f7bf-bebf-48ac-a25c-8d75e08fc62f)

2. No "Select All" on folder page.
![image](https://github.com/grafana/grafana/assets/53630996/f1395e01-14ff-462c-a9e9-a9ebd6b2079a)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82701. Feel free to continue the discussion around this there. Thank you!
